core/src/main/java/discord4j/core/object/entity/Guild.java

```diff
@@ -544,15 +544,12 @@ public Mono<GuildEmoji> getGuildEmojiById(final Snowflake id, EntityRetrievalStr
     }
 
     /**
-     * Gets the enabled guild features.
-     * <br>
-     * You can see the available
-     * <a href="https://discord.com/developers/docs/resources/guild#guild-object-guild-features">guild features</a>
+     * Returns the enabled features of this {@link Guild}.
      *
-     * @return The enabled guild features.
+     * @return A {@code EnumSet} with the enabled guild features.
      */
-    public Set<String> getFeatures() {
-        return new HashSet<>(data.features());
+    public EnumSet<FeatureSet> getFeatures() {
```

Comment:
This need test... if this is a minor update the method in teory cannot be broken unless this change not break people using this method.